plugins/ffmpeg_plugin/README.md

```diff
@@ -0,0 +1,18 @@
+# The kahawai ffmpeg plugin
```

Comment:
Pls help to remove the old ffmpeg_plugin.md.

Reply:
Done and thanks:-)